src/matter/cluster/GroupsCluster.ts

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    Field,
+    ObjectT,
+    StringT,
+    UInt8T,
+    ArrayT,
+    EnumT, BitMapT, Bit
+} from "../../codec/TlvObjectCodec";
+import { Attribute, Cluster, Command, NoArgumentsT, NoResponseT } from "./Cluster";
+import { StatusCode } from "../interaction/InteractionMessages";
+import { GroupIdT } from "../common/GroupId";
+import { MatterApplicationClusterSpecificationV1_0 } from "../../Specifications";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.3.7.1 */
+const AddGroupRequestT = ObjectT({
+    groupId: Field(0, GroupIdT), /* min: 1 */
```

Comment:
Should "min: 1" be part of GroupId ?
Or has GroupId = 0 a special meaning?

Reply:
bit I add min as Bound now

Reply:
Ok, let's keep it like that and fix it later when we rework Templates